src/games/_gameswidget.py

```diff
@@ -12,13 +12,23 @@
 import fa
 import modvault
 import notifications as ns
+from config import Settings
 
 import logging
+
 logger = logging.getLogger(__name__)
 
 FormClass, BaseClass = util.loadUiType("games/games.ui")
 
+PLAY_SUBSET_FACTIONS = "play/subsetFactions"
+PLAY_SUBSET_SELECT = "play/selectSubset"
+
+
 class GamesWidget(FormClass, BaseClass):
+
+    Settings.persisted_property(PLAY_SUBSET_SELECT, type=bool, default_value=False)
+    Settings.persisted_property(PLAY_SUBSET_FACTIONS, default_value=[False, False, False, False, False])
```

Comment:
Ditto on this (As here: https://github.com/FAForever/client/pull/318#discussion-diff-57013181), you'll want to assign these properties for intended use.